lisa-utils/src/main/scala/lisa/utils/LisaException.scala

```diff
@@ -53,7 +56,7 @@ object UserLisaException {
       Console.RED + proof.owningTheorem.repr + Console.BLACK + "\n" +
         ProofPrinter.prettyProof(proof, 2) + "\n" +
         "  " * (1 + proof.depth) + Console.RED + textline + Console.BLACK + "\n\n" +
-        s"   Proof tactic ${tactic.name} used in.(${file.value.split("/").last}:${line.value}) did not succeed:\n" +
+        s"   Proof tactic ${tactic.name} used in.(${file.value.split("/").last.split("\\\\").last}:${line.value}) did not succeed:\n" +
```

Comment:
`in.( -> in (`

I deal with this message far too often to not notice it everyday haha!

Reply:
Hahaha :D
Actually, for some reason, it is needed to become a clickable link that leads you to the file and code line. Don't ask me why though.

Reply:
HUH :thinking: 

well fine I guess lol